src/status_im/utils/universal_links/core.cljs

```diff
@@ -109,8 +109,9 @@
   (log/info "universal-links: handling view profile" public-key)
   (cond
     (and public-key (new-chat.db/own-public-key? db public-key))
-    {:shell/change-tab-fx :browser-stack ;; Profile tab - Currently browser tab is used for profile
-     :pop-to-root-fx      :shell-stack}
+    (rf/merge cofx
+              {:pop-to-root-fx :shell-stack}
+              (navigation/navigate-to-cofx :my-profile nil))
```

Comment:
How are we sure that there won't be a race condition between these two navigation actions?

Reply:
cc: @flexsurfer ?

Reply:
good question, in theory it might happen, because the effect is a map, and no guarantee it will be processed in order how it's in `fx/merge` macro, but this never happened, so just luck :)